src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -905,6 +905,9 @@ private void BuildSymbolsPackage(string path)
             {
                 symbolsBuilder.PackageTypes.Clear();
                 symbolsBuilder.PackageTypes.Add(PackageType.SymbolsPackage);
+                symbolsBuilder.PackageAssemblyReferences.Clear();
+                // Remove the references when building the symbols package.
+                // They are not relevant for the symbols packages (snupkgs specifically).
```

Comment:
comments are usually above the code they refer to. Any particular reason to break convention here?

Reply:
I can move it above.